css/properties/text-size-adjust.json

```diff
@@ -21,16 +21,10 @@
             "chrome_android": {
               "version_added": "54"
             },
-            "edge": [
-              {
-                "prefix": "-ms-",
-                "version_added": true
-              },
-              {
+            "edge": {
                 "prefix": "-webkit-",
-                "version_added": true
-              }
-            ],
+                "version_added": "12"
```

Comment:
Wrong indentaion, it should be (ie. remove two spaces before `version_added` and `prefix`):
```json
            "edge": {
              "version_added": "12",
              "prefix": "-webkit-"
            },
```